docs/dsl_resource.md

```diff
@@ -98,3 +98,29 @@ end
 ```
 
 For a full example, see our [example resource](https://github.com/chef/inspec/blob/master/examples/profile/libraries/example_config.rb).
+
+## Lazy Loading
+
+Prior to InSpec v4.16, resources were pre-loaded for every invocation
+of `inspec`. This was a heavy and unnecessary burden on the system and
+exacerbated startup times (especially on Windows).
+
+As of InSpec v4.16, resources are lazily loaded into the `inspec`
+process upon use. This greatly speeds up the initial startup costs of
+the `inspec` process and only loads what you need to use (eg, `inspec
+--version` no longer runs for 10 seconds!).
```

Comment:
```suggestion
--version` no longer runs for 10 seconds!.
```

Reply:
no. see the `eg` on the line before. this is totally a parenthetical.

Reply:
According to our internal style guides, "e.g." is not supposed to be used in documentation. Please use "for example," or something not in Latin.